docs/index.rst

```diff
@@ -18,13 +18,13 @@ customer datacenters.
 .. _Scality: https://www.scality.com
 .. _Zenko: https://www.zenko.io
 
-It is based on the Kubespray_ project to reliably install a base Kubernetes
+MetalK8s, based on the Kubespray_ project, reliably installs a base Kubernetes
 cluster, including all dependencies (like etcd_), using the Ansible_
-provisioning tool. This installation is further augmented with operational
-tools for monitoring and metering, including Prometheus_, Grafana_,
-ElasticSearch_ and Kibana_. Furthermore, an "ingress controller" is deployed
-by default, based on Nginx_. All of these are managed as Helm_ packages. See
-:doc:`architecture/cluster-services` for a whole listing.
+provisioning tool. MetalK8s augments this installation with operational
+monitoring and metering tools, including Prometheus_, Grafana_, ElasticSearch_
```

Comment:
No, they're not "operational monitoring and metering tools". They're tool for operational purposes, including monitoring and metering.

Reply:
The logic of the original sentence is as follows:
* There exists an installation.
* The installation is augmented.
* The thing augmenting the installation is a set. 
* This set is described as "operational tools for metering and monitoring".
* This set contains: Prometheus, Gafana, Elasticsearch, and Kibana.
 
As I read your response, the second-to last point is where the error crept in. If this assessment is accurate, my suggestion is to unpack this phrase. If I read you correctly, the sentence should be: 

"MetalK8s augments this installation with operational tools, including Prometheus, [etc...], for monitoring, metering, and other purposes."

Reply:
Fair enough wrt the last suggestion.